examples/hat/binaryclock.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+import unicornhat as uh
+import datetime
+from time import sleep
+
+print("""Binar Clock made by Iorga Dragos Florian
+Displays Hour(red), Minute(yellow), Second(blue), Day(green) and Month(Pink) in binary format""")
+FPS = 30.0
+uh.rotation(0)
+uh.brightness(1)
+
+def draw_time(b,x,idx,color):
+    if b == '1':
+        uh.set_pixel(x,idx,color)
+    sleep(1.0/FPS)
```

Comment:
This will drop your framerate to `FPS / number_of_times_draw_time_is_called`

The sleep should go at the bottom of your `while` loop after all drawing operations have completed.

Reply:
I'm scrapping this whole FPS thing because it makes the hat flash which is annoying, mine is much smoother so far, as soon as done will share.